### DISCUSSION TITLE
Espressopp weekly meeting (4.30pm, 7. Jul)

### DISCUSSION THREAD

**Discussion by xzhh** [espressopp/espressopp#457] [Team Posts]
Is correct to choose private members?

> Join Zoom Meeting
> https://zoom.us/j/92605124006?pwd=c1dlR3JDUHlIYjJXeEVIcGlLaGc1QT09
>
> Meeting-ID: 926 0512 4006
> Kenncode: 120525

**Comment by XzzX** [comment]
I got an email :+1:

**Comment by junghans** [comment]
+1
